### PR TITLE
[OBSDEF-4220] Updated versions for clr-icons and clr-ui to 4.0.8

### PR DESCRIPTION
**Pull request by akate1**
**Description:**
Updated and tested `clr/ui` and` clr/icons `packages to version 4.0.x

**JIRA:**
- https://jira.cec.lab.emc.com/browse/OBSDEF-4220

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     "description": "React components for Clarity UI",
     "license": "Apache-2.0",
     "private": false,
     "outDir": "dist",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "dependencies": {
-        "@clr/icons": "3.0.0",
-        "@clr/ui": "3.0.0",
+        "@clr/icons": "4.0.8",
```
Comment (payald) [inline package.json:11]: This is not related to this PR. Same changes are needed in vsphere UI repo.
"@clr/icons": "4.0.8",
"@clr/ui": "4.0.8",

**Review by payald — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (akate1) [reply package.json:11]: @payald We are going to take care of this in different PR once this PR gets merged

**Review by girishdudhe — APPROVED** [review]
LGTM

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM
